react/package.json

```diff
@@ -17,9 +17,8 @@
     "react-infinite-scroll-component": "^4.1.0",
     "react-intl": "^2.4.0",
     "react-motion": "^0.5.2",
-    "unorm": "^1.4.1",
-    "react-apollo": "^2.1.3",
-    "react-intl": "^2.4.0"
+    "react-outside-click-handler": "^1.2.2",
```

Comment:
Why was `react-apollo` and `react-intl` removed?

Reply:
There were not, i think they were updated and it is showing like they were removed